### PR TITLE
Seperate relayer instance in integration tests

### LINKED ISSUE #95: Update README.md environment variables and query parameters

### LINKED ISSUE #147: Make integration tests run their own instance of the relayer

### LINKED ISSUE #149: Update the `seedDb` script

### PR DESCRIPTION
**Pull request by simzzz**
closes #147 closes #149 closes #95

### CODE REVIEW

**Comment by samparsky** [comment]
@simzzz Does this enable us to run the commented out integration tests? i.e. `postMedia` etc

**Comment by simzzz** [comment]
> @simzzz Does this enable us to run the commented out integration tests? i.e. `postMedia` etc

Our IPFS node is IP whitelisted, which makes the tests unreliable. That's why I commented out the tests that use it.

File: routes/adSlots.js [hunk 1]
```diff
@@ -341,7 +340,7 @@ async function verifyWebsite(req, res) {
 	}
 }
 
-async function getTargetingData(req, res) {
```
Comment (Ivshti) [inline routes/adSlots.js:344]: not sure this line would work without `req, `

**Review by Ivshti — CHANGES_REQUESTED** [review]
Looks good except one missed thing which might break things @simzzz

**Pull request merged**